web/modules/custom/shepherd/shp_orchestration/src/Plugin/OrchestrationProvider/OpenShiftOrchestrationProvider.php

```
@@ -986,4 +1010,44 @@ private function handleClientException(ClientException $exception) {
     drupal_set_message(t("An error occurred while communicating with OpenShift. %reason", ['%reason' => $reason]), 'error');
   }
 
+  /**
+   * @param $deployment_name
+   * @param $source_ref
+   * @param $cron_suspended
+   * @param $cron_jobs
+   * @param $image_stream
+   * @param $volumes
+   * @param $deploy_data
+   *
+   * @return bool
+   */
+  private function createCronJobs(string $deployment_name, string $source_ref, bool $cron_suspended, array $cron_jobs, array $image_stream, array $volumes, array $deploy_data) {
+    if ($image_stream) {
+      foreach ($cron_jobs as $schedule => $args) {
+        $args_array = [
+          '/bin/sh',
+          '-c',
+          $args,
+        ];
+        try {
+          $this->client->createCronJob(
+            $deployment_name . '-' . \Drupal::service('shp_custom.random_string')->generate(5),
+            $image_stream['status']['dockerImageRepository'] . ':' . $source_ref,
+            $schedule,
+            $cron_suspended,
+            $args_array,
+            $volumes,
+            $deploy_data
+          );
+        }
+        catch (ClientException $e) {
+          $this->handleClientException($e);
+          return FALSE;
+        }
+      }
+    }
+
+    return TRUE;
```

Silent fail if no image stream.